### PR TITLE
Scoped packages support

### PR DESCRIPTION
**Pull request by vdeturckheim**
add support for scoped packages ([see npm doc](https://docs.npmjs.com/misc/scope))
add node 6 as travis target

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -6,11 +6,14 @@ module.exports = function (file) {
   var segments = file.split(path.sep)
   var index = segments.lastIndexOf('node_modules')
   if (index === -1) return
-  var name = segments[index + 1]
+  if (!segments[index + 1]) return
+  var scoped = segments[index + 1].indexOf('@') === 0
```
Comment (vdeturckheim) [inline index.js:10]: npm refuses names that start with '@' except in the case of scoped packages

**Comment by watson** [comment]
@vdeturckheim Thanks! I did not think of this. Really happy for your help 👍 

Regarding the addition of Node.js v6 to `.travis.yml` that's really awesome as well, but it should technically have been a separate commit. Could I ask you do simply rebase this commit to split it up into two commits with the `.travis.yml` update being in one and the rest being in another? Then just force push to the same branch which will update this PR. Thanks 😃

**Comment by vdeturckheim** [comment]
@watson : no worries !
Here are the two commits ;)

**Comment by watson** [comment]
@vdeturckheim thanks 😃 It's released as v1.0.3 on npm

**Pull request merged**